src/Symfony/Component/Yaml/Inline.php

```diff
@@ -127,6 +127,8 @@ public static function dump($value, int $flags = 0): string
                 return self::dumpNull($flags);
             case $value instanceof \DateTimeInterface:
                 return $value->format('c');
+            case $value instanceof \UnitEnum:
```

Comment:
Do we need to adjust the DI component's composer.json

Reply:
I don't think so, the current `composer.json` seems good with `"^4.4|^5.0"`